common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -3510,6 +3510,63 @@ typedef int (*evp_aead_ctx_op_func)(const EVP_AEAD_CTX* ctx, uint8_t* out, size_
                                     const uint8_t* in, size_t in_len, const uint8_t* ad,
                                     size_t ad_len);
 
+static jint evp_aead_ctx_op_ssl(JNIEnv* env, jlong evpAeadRef, jbyteArray keyArray, jint tagLen,
```

Comment:
nit: maybe `evp_aead_ctx_op_common`, but I don't really have a strong opinion :)

Reply:
Common works! I just picked ssl since it was doing the final ssl function but makes no difference to me